### ISSUE TITLE
tf.layers.dropout()'s rate is not keep_prob

### ISSUE THREAD

**Issue by wizardbc** [hunkim/DeepLearningZeroToAll#204]
In `lab-11-4-mnist_cnn_layers.py` and others using the `tf.layers.dropout()`,
dropout `rate` is not the same as `keep_prob`. Actually, dropout `rate`=1-`keep_prob`.

So `dropout_rate` should be 0.3 not 0.7.

Thank you all for this great lectures and examples. I'm really enjoying  it. :)

**Comment by kkweon** [comment]
Duplicate #122

**Comment by kkweon** [comment]
@wizardbc Could you please send a PR to fix this?

**Comment by kkweon** [comment]
fixed with #206 #205
